test/test_dev.py

```diff
@@ -5,13 +5,12 @@
                         unicode_literals)
 
 import os
-import sys
 import re
 from os.path import abspath, dirname, join, relpath
 import shutil
 import pytest
 
-import six
+import six  # noqa
```

Comment:
Why is this? In case it's really needed you need the error code to skip and the reason.